js/friday.js

```diff
@@ -1,34 +1,38 @@
 (function IsItFridayYet() {
   const date = new Date();
-  const theMonth = date.getMonth();
-  const theDay = date.getDay();
-  const element = document.getElementById("friday");
-  const spans = document.getElementsByClassName("fallingElement");
+  const thisMonth = date.getMonth();
+  const thisDay = date.getDay();
+  const theBody = document.body;
+  const textElement = document.getElementById("friday");
   let season = "";
 
-  if (theDay === 5) {
-    element.innerHTML = "YES";
+  // return a programatic yes on Fridays to print on homepage.
```

Comment:
This function will expect a number. If the number is 5, we get a yes, anything else will always be a no. Since our function is super simple, we can use a ternary operator to output our condition.

**Now get ready for some fun JS controversy.** 

Marvin, you monster. `n == 5` Why would you double equal? Are you mad?

YES. Mad and confident. Here is how the double equals works:
1. Check if the type comparison is of the same type. If they are, auto-preform a triple equal.
2. if it is not the same type, coerce until true.

The moral of the story and one that will likely have me stoned to death on some forums is the thing I want to instill into you: Be confident in your types and in the way you write JavaScript. **I KNOW** that when I invoke this function that it will pass a number. I know that the comparison will only ever be a number. I only should ever really think about a `===` if I wasn't sure... _And if I'm not sure_, then there is a bigger problem that needs to be addressed and it's not arguing over double or triple equals.

You are the master of your types. Don't let anyone else tell you otherwise.